modules/seqexec/web/server/src/main/scala/seqexec/web/server/http4s/WebServerLauncher.scala

```diff
@@ -137,7 +137,7 @@ object WebServerLauncher extends IOApp with LogInitialization {
              /_/
 """
     val msg = s"""Start web server for site ${conf.site} on ${conf.mode} mode, version ${OcsBuildInfo.version}"""
-    L.info(banner + msg)
+    Logger[F].info(banner + msg)
```

Comment:
Why this change? Doesn't `L` still exist in this context?

Reply:
I thought about that, L is `Logger[IO]` and I wanted `Logger[F]`